### DISCUSSION TITLE
Custom httpd/php 8 environment not working?

### DISCUSSION THREAD

**Discussion by TorutheRedFox** [Rehike/Rehike#363] [General]
![image](https://github.com/Rehike/Rehike/assets/15708278/200545b5-91dc-4118-b359-c71a3275d121)
I am having issues getting it to make any connection in my own existing (and otherwise working, see https://www.neoalpha.net/) environment

Is there anything specific I may need in my httpd/php config to make this work?

**Comment by ephemeralViolette** [comment]
Completely unsure. These are issues with outgoing requests from the local Rehike server to the remote YouTube server, but cURL seems to be available.

I can replicate this exact same style of error (response code 0, no further information) by blocking httpd from making outgoing requests in my firewall. That's the first place I would check.

**Comment by TorutheRedFox** [comment]
things is I've managed to make requests myself using cURL in PHP for my _own_ hitchhiker remake thing, albeit to a self-hosted Invidious instance and it works (I used someone else's Invidious instance to make a proof of concept before setting my own up and it worked just fine too)

there's nothing in the firewall either, which is odd

**Comment by TorutheRedFox** [comment]
(my stuff is running off the same httpd instance)

**Comment by TorutheRedFox** [comment]
I fixed it
turns out my cacert config was bad
